pubnub-kotlin/pubnub-kotlin-impl/src/integrationTest/kotlin/com/pubnub/api/integration/LargeMessagePublishIntegrationTests.kt

```
@@ -0,0 +1,243 @@
+        val largeMessage = "x".repeat(50_000)
+        val success = AtomicBoolean()
+
+        val observer = createPubNub {}
+
+        pubnub.addListener(
+            object : SubscribeCallback() {
+                override fun status(
+                    pubnub: PubNub,
+                    pnStatus: PNStatus,
+                ) {
+                }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)